pkgs/by-name/gi/github-desktop/package.nix

```diff
@@ -91,6 +96,13 @@ stdenv.mkDerivation (finalAttrs: {
     yarnOfflineCache="$cacheApp" yarnConfigHook
     popd
 
+    # This is insanely hacky, I don't even really understand why this has to be
+    # the way it is...
+    pushd app/node_modules/desktop-notifications
+    yarnOfflineCache="$cacheDesktopNotifications" yarnConfigHook
+    npm_config_nodedir="${nodejs}" yarn --offline install
+    popd
+
```

Comment:
Uhh, I think the way I did it before the version bump would also work just fine... Going to bed first 😅